### PR TITLE
Create readme file

### PR DESCRIPTION
**Pull request by AngeloCG97**

### CODE REVIEW

**Comment by xavier506** [comment]
@AngeloCG97 

- Is the LAC Chain Logo showing?

- Lets adjust the file structure and add description texts to missing files / folders 

image

- EOS Costa Rica Logo is not centered

File: README.md [hunk 1]
```diff
@@ -1,9 +1,79 @@
...
+│   └── holacontrato.sh .................. Se muestra cómo compilar para el contrato holacontrato
+├── prepareTransaction
+│   └── eosJS
+│       └── holacontrato.js .................. Se muestra cómo preparar una transacción para el contrato holacontrato por medio de eosJS
+│       └── transacction.js .................. Se muestra cómo preparar una transacción por medio de eosJS
+│	└── shellScript
+│		└── holacontrato.sh .................. Se muestra cómo preparar una transacción para el contrato holacontrato por medio de sh
+├── LICENSE .................. Términos y Condiciones
+└── README.md ................ Especificación del repositorio
+```
```
Comment (leisterfrancisco) [inline README.md]: What if we make it like this:
```
/
├── deployContract
│   └── holacontrato.cpp ..................... Se muestra el contrato holacontrato que se encuentra en la red https://lacchain.eosio.online/accounts
│   └── holacontrato.sh ...................... Se muestra cómo compilar para el contrato holacontrato
├── prepareTransaction
│   └── eosJS
│       └── holacontrato.js .................. Se muestra cómo preparar una transacción para el contrato holacontrato por medio de eosJS
│       └── transacction.js .................. Se muestra cómo preparar una transacción por medio de eosJS
│	└── shellScript
│		└── holacontrato.sh .......... Se muestra cómo preparar una transacción para el contrato holacontrato por medio de sh
├── LICENSE .................................. Términos y Condiciones
└── README.md ................................ Especificación del repositorio
```

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,9 +1,79 @@
+## Lacchain-scripts
```
Comment (xavier506) [inline README.md]: maybe we dont need this header

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**